src/common/actions/repositories.js

```diff
@@ -35,14 +37,21 @@ function checkStatus(response) {
   }
 }
 
-export function fetchUserRepositories() {
+export function fetchUserRepositories(pageNumber) {
   return (dispatch) => {
+    let urlParts = [BASE_URL, 'api/github/repos'];
+
+    if (pageNumber) {
+      urlParts.push('page/' + pageNumber);
+    }
 
     dispatch({
       type: FETCH_REPOSITORIES
     });
 
-    return fetch(`${BASE_URL}/api/github/repos`, {
+    dispatch(setRepositories([]));
```

Comment:
Clear repositories when page is changed. This allows the spinner to be shown without making the repo list jump down.